app/scripts/lib/channels/notifier.js

```diff
@@ -20,12 +20,12 @@ define(function (require, exports, module) {
     CHANGE_PASSWORD: {
       name: 'fxaccounts:change_password',
       schema: {
-        email: 'String',
-        keyFetchToken: 'String',
-        sessionToken: 'String',
-        uid: 'String',
-        unwrapBKey: 'String',
-        verified: 'Boolean'
+        email: Vat.email().required(),
+        keyFetchToken: Vat.keyFetchToken().optional(),
```

Comment:
If I apply the following patch to ensure each field in the schema is validated, functional tests start failing all over the place:

```diff
diff --git a/app/scripts/lib/validate.js b/app/scripts/lib/validate.js
index 82606571..d7e49b05 100644
--- a/app/scripts/lib/validate.js
+++ b/app/scripts/lib/validate.js
@@ -178,7 +178,7 @@ define(function (require, exports, module) {
         return false;
       }
 
-      return _.all(_.keys(data), function (key) {
+      return _.all(_.keys(schema), function (key) {
         return isDataValid(data[key], schema[key]);
       });
     },
```

What is really needed is the union of _.keys(data) and _.keys(schema) to ensure each field in both the data and schema are validated.